Remove OBS label from breadcrumbs

bgeuken opened the pull request:
On each page the breadcrumb starts with 'openSUSE Build Service'.
This text is taking away quite some space from the breadcrumbs and it
is not adding much value. Users already know that they are looking at
the OBS when they reached this page.
In case they don't we still have the OBS logo in the navbar.

The link to the main page is now part of the 'home' icon that is
part of each breadcrumb.

-------------------

Depending on which page you are this looks a bit different. That's why I added a couple of screenshots:

On the main page:

![selection_004](https://user-images.githubusercontent.com/968949/50007859-97bfef80-ffb2-11e8-8a7b-a84af7f35b3e.png)


On the other pages:

![selection_005](https://user-images.githubusercontent.com/968949/50007868-a0b0c100-ffb2-11e8-8a33-73828433c39d.png)

![selection_006](https://user-images.githubusercontent.com/968949/50007876-a5757500-ffb2-11e8-8fe4-bf88d8964407.png)

dmarcoux commented on src/api/app/views/webui2/webui/main/_breadcrumb_items.html.haml:
```diff
@@ -5,5 +5,5 @@
   %li.breadcrumb-item
-    %i.fas.fa-home.mr-1
-    = link_to home_title, root_path
+    = link_to root_path do
```
I would add a title here, perhaps `'Homepage'` or `home_title`

bgeuken replied:
The whole point was to not add a title here. Though we could add a tooltip.

dmarcoux replied:
Sorry I wasn't clear, I was referring to the `title` attribute of the `a` tag generated by `link_to`

eduardoj replied:
I'm fine with @bgeuken's approach. I would add the tooltip.

dmarcoux commented on src/api/app/views/webui2/webui/main/_breadcrumb_items.html.haml:
```diff
@@ -5,5 +5,5 @@
     = home_title
```
It's somewhat weird now, because on the homepage, this is what we see:

![homepage](https://user-images.githubusercontent.com/1102934/49881200-13446400-fe2e-11e8-92a3-a032c0cb1fa0.png)

And on the other pages:
![project-show](https://user-images.githubusercontent.com/1102934/49881192-0fb0dd00-fe2e-11e8-9f96-92da1ff85b86.png)

I think `home_title` should also be removed here. There should also be a title as I said in my other comment.

bgeuken replied:
I added some reasoning in my other comment. I wanted to get some feedback first before changing this further.

bgeuken commented:
As you can see on the main page the 'Open Build Service' label is right next to the 'house' icon, while on the other pages the 'house' and the rest of the breadcrumb is divided by the '/' sign we use for dividing the breadcrumb parts.

IMO this is fine, since the house is the main page. Thus we don't have to divide the two elements on the main page, while on the other pages they represent different things.

@openSUSE/obs-frontend What do you think?

DavidKang commented:
@bgeuken, and if we remove the breadcrumb only on the main page?

bgeuken commented:
@DavidKang @dmarcoux @Ana06 Updated and I removed the breadcrumb from the main page as suggested by David. I've updated the screenshots as well. Have a look
